src/main/resources/application.yaml

```diff
@@ -0,0 +1,9 @@
+server:
+  port: 8080
+
+minimumAgeOfEmployee: 18
+
+spring:
```

Comment:
Please don't define profile in application.yaml. You need to set it as env variable. Please look at this link:
https://stackoverflow.com/questions/38520638/how-to-set-spring-profile-from-system-variable
-Dspring.profiles.active=h2 - example to use in VM options